src/PostgREST/Config.hs

```diff
@@ -460,3 +462,34 @@ type Environment = M.Map [Char] Text
 readPGRSTEnvironment :: IO Environment
 readPGRSTEnvironment =
   M.map T.pack . M.fromList . filter (isPrefixOf "PGRST_" . fst) <$> getEnvironment
+
+-- | Adds a `fallback_application_name` value to the connection string. This allows querying the PostgREST version on pg_stat_activity.
+--
+-- >>> let ver = "11.1.0 (5a04ec7)"::ByteString
+--
+-- >>> addFallbackAppName ver "postgres://user:pass@host:5432/postgres"
+-- "postgres://user:pass@host:5432/postgres?fallback_application_name=PostgREST%20..."
```

Comment:
Btw, I've noticed these `...` work as a wildcard. Do you have a link for this behavior?

Reply:
Yes, found it in their repo's README: https://github.com/sol/doctest#matching-arbitrary-output